src/rules/pylint/settings.rs

```diff
@@ -53,17 +53,28 @@ pub struct Options {
     )]
     /// Constant types to ignore when used as "magic values".
     pub allow_magic_value_types: Option<Vec<ConstantType>>,
+    #[option(
+        default = r#"5"#,
+        value_type = "usize",
+        example = r#"
+            allow-magic-value-types = 5
+        "#
+    )]
+    /// Constant types to ignore when used as "magic values".
```

Comment:
Looks like this was copied over from `allow_magic_value_types` -- mind writing something relevant for this new option?